### ISSUE TITLE
[1.4.0] Start-AzAutomationRunbook is converting the json parameters to clixml

### ISSUE THREAD

**Issue by sridharktm** [Azure/azure-powershell#13240]
When I execute a runbook using the this powershell cmdlet , my automation runbooks fails as the input parameters are being submitted as CliXml instead of json. Before submitting the job, I confirmed the object type i am passing as parameter is idictionary.
This I am experiencing only when using 1.4.0 az.automation module.

$json = (Get-content -path $Parametersfile -Raw) | Out-string
$params = @{"json"=$json}
Start-AzAutomationRunbook -AutomationAccountName $AutomationAcct -ResourceGroupName $resourcegroup -Name $RunbookName -Parameters $params

**Comment by dingmeng-xue** [comment]
Service team, please help to look into this question.

**Comment by larsthelord** [comment]
This happened to me as well when passing Json-strings across workbooks. 

I created a workaround in case anyone needs it.
The Json string is passed between runbooks as a hashtable containing the CliXml containing your Json String. 

``` powershell
$CliXmlFile = New-TemporaryFile 
Set-Content -Path $CliXmlFile.FullName -Value $<JsonVariable>.CliXml
$JsonContent = (Import-CliXml -Path $CliXmlFile.FullName) | ConvertFrom-Json
```

**Comment by larsthelord** [comment]
Additional information:
While downgrading the module (It also screwed up string encoding, causing me more troubles) I first downgraded to 1.3.7
The same issue happened here.
1.3.6 acted as is had been doing previously. 
Release note for 1.3.7 is "Fixed the issue that string with escape chars cannot be converted into json object."
Perhaps this is where the breaking change is.

**Comment by Mattheww65** [comment]
I am also experiencing this issue - thankyou @larsthelord
Rolling back to 1.3.6 didn't work for me - see attached for the JSON conversion
![az2](https://user-images.githubusercontent.com/73503344/97236163-16d6ff00-17aa-11eb-8a09-a5e8ddd1e839.JPG)

![az1](https://user-images.githubusercontent.com/73503344/97236070-ed1dd800-17a9-11eb-9fd4-9f3773fb1c22.JPG)

**Comment by jaspkaur28** [comment]
@sridharktm @Mattheww65 @larsthelord Can you pls share more details : error message , jobId, automation account ID etc , so we can check out backend on the failure. Would you great if one of you can share the sample runbook code for us to debug .

**Comment by RationalSquash** [comment]
Having the exact same issue with V 1.4.0

Didn't get the tempfile workaround to work, so i made my own for the time being

```powershell
if($JSONString.Substring(2,6) -eq "CliXml"){
    $JSONString= [Management.Automation.PSSerializer]::Deserialize(($JSONString.Substring(0,$JSONString.length-1)).Substring(9))
}
```

I don't know if this will work for large and complex JSON input, but for my case with simple nested lists, this works perfectly.

**Comment by jaspkaur28** [comment]
we are seeing the issue due to this commit : 
https://github.com/Azure/azure-powershell/commit/9562911d02e19978cb668f1e3ea0f55cd16862b4#diff-415dc886667885776ccc3a94b73722653b94d33be77a8421620c8522e1f20e62 

Have looped in the Powershell team to look into this.

**Comment by dingmeng-xue** [comment]
For this case, `Out-String` should be removed because it accumulates the strings and wrapped as PSObject. Actually, it is different from original string. This constraints was introduced when we fixed #11504. We will update Automation module by the end of this year and make code support this case.

$json = (Get-content -path $Parametersfile -Raw) **| Out-string**
